src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/PrefetchGetRecordsCache.java

```diff
@@ -53,13 +55,14 @@ private void start() {
     }
 
     @Override
-    public GetRecordsResult getNextResult() {
+    public ProcessRecordsInput getNextResult() {
         if (!started) {
```

Comment:
I'm not a huge fan of automatically starting like this.  I prefer to require that start is called or an `IllegalStateException` is thrown.  Though it may not make much of a difference in this case.